src/main/java/com/scalyr/integrations/kafka/AddEventsClient.java

```diff
@@ -92,6 +93,20 @@
    */
   private static final AddEventsResponse TIMEOUT_ADD_EVENTS_RESPONSE = new AddEventsResponse().setStatus("Timeout").setMessage("Timeout");
 
+  /** Add events API has max payload size of 6 MB */
+  private static final int MAX_ADD_EVENTS_PAYLOAD_BYTES = 6000000;
+
+  /**
+   * Response when an add events request exceeds MAX_ADD_EVENTS_PAYLOAD_BYTES.  We skip add event requests > 6 MB
+   * since it will be rejected by the server.  Success is returned to the caller so it won't be retried.
+   */
+  private static final AddEventsResponse PAYLOAD_TOO_LARGE = new AddEventsResponse().setStatus(AddEventsResponse.SUCCESS).setMessage("Skipped due to payload too large");
```

Comment:
Add a TODO to make this configurable.

Reply:
Are you thinking the status or message should be configurable?